apps/frontend/src/extensions/SlashCommand/MenuList.tsx

```diff
@@ -118,13 +119,16 @@ export const MenuList = React.forwardRef((props: MenuListProps, ref) => {
   return (
     <Surface
       ref={scrollContainer}
-      className="mb-8 max-h-[min(80vh,24rem)] overflow-auto p-2"
+      className="mb-8 max-h-[min(80vh,24rem)] overflow-auto p-2 dark:bg-background border-[#3A3A3A]"
```

Comment:
```suggestion
      className="mb-8 max-h-[min(80vh,24rem)] overflow-auto p-2 bg-background"
```
we can use this color variable "bg-background" to apply the background. it will be applied both light and dark mode.
then we can go to Surface component on Surface.tsx and remove the classes "bg-white" and "dark:bg-neutral-900", as I showed in the suggestion there in the component.